cmd/operator-sdk/bundle/cmd.go

```diff
@@ -39,16 +39,18 @@ native software, like the Operator Lifecycle Manager.
 
 More information about operator bundles and metadata:
 https://github.com/operator-framework/operator-registry#manifest-format.
-
-More information about the integration with OLM via SDK:
-https://sdk.operatorframework.io/docs/olm-integration/
 `,
 	}
 	return cmd
 }
 
 func NewCmdLegacy() *cobra.Command {
 	cmd := newCmd()
+	// TODO(estroz): uncomment this once OLM integration docs are updated.
+	// 	cmd.Long += `
+	// More information about the integration with OLM via SDK:
+	// https://sdk.operatorframework.io/docs/olm-integration/legacy
+	// `
```

Comment:
This can't merge uncommented with this PR?

Reply:
No, the link check was unhappy.